sormas-backend/src/main/java/de/symeda/sormas/backend/event/Event.java

```diff
@@ -304,6 +310,32 @@ public void setMeansOfTransportDetails(String meansOfTransportDetails) {
 		this.meansOfTransportDetails = meansOfTransportDetails;
 	}
 
+	@Column(columnDefinition = "text")
```

Comment:
are you expecting here string of unlimited size ? i think it would be better to define it as varchar with limited size (the current default in sormas is 512)

Reply:
updated